pyproject.toml

```diff
@@ -53,7 +53,7 @@ dependencies = [
 [project.optional-dependencies]
 tf = [
     # cf. https://github.com/mindee/doctr/pull/1461
-    "tensorflow>=2.15.0,<3.0.0",
+    "tensorflow[and-cuda]>=2.15.0,<3.0.0",
```

Comment:
Mh it looks like that's conflicting with MacOS unfortunately :/  --> failing CI job

Reply:
We could try to control it with sys_platform condition -> https://peps.python.org/pep-0631/#example

Reply:
something like:
```
"tensorflow[and-cuda]>=2.15.0,<3.0.0; platform_system != 'Darwin' or tensorflow>=2.15.0,<3.0.0; platform_system == 'Darwin'"
```
Or (not sure that the `or` condition works correctly

```
"tensorflow[and-cuda]>=2.15.0,<3.0.0; platform_system != 'Darwin'",
# TensorFlow compatible for macOS
"tensorflow>=2.15.0,<3.0.0; platform_system == 'Darwin'",
```

Reply:
works ^^ 👍🏼